internal/clientio/requestparser/resp/respparser.go

```diff
@@ -0,0 +1,249 @@
+package respparser
```

Comment:
Just a question, will this replace the other resp parser at `internal/clientio/resp.go`?